packages/wrangler/templates/facade.d.ts

```diff
@@ -1,9 +1,24 @@
+// TODO(soon): remove once https://github.com/cloudflare/workerd/pull/1870 lands
```

Comment:
Eagle has landed 🦅

Just need to publish to npm, and then we should be good to go. Looks like there was a problem with that latest run though: https://github.com/cloudflare/workerd/actions/runs/8438068332

Reply:
Are we going to remove this file before merging?

Reply:
Thanks for the reminder. Yes, I'll do that now.